substrate-bip39/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "substrate-bip39"
+version = "0.3.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+license = "GPL-3.0"
+edition = "2018"
+description = "Converting BIP39 entropy to valid Substrate (sr25519) SecretKeys"
```

Comment:
Should we indicate somewhere that this version 0.3.1 has modifications and is not the same as the "official" 0.3.1? From the perspective of a user who could simply say: "Oh, it's the same version, let's take the official"

Reply:
done in cl-auto-config-from-metadata